src/modules/shared/components/PaginationBar/PaginationBar.module.scss

```diff
@@ -43,14 +43,13 @@ $component: "c-pagination-bar";
 	grid-template-columns: repeat(2, 1fr);
 	grid-template-rows: min-content;
 	align-items: center;
-	justify-items: center;
 
 	&:not(.#{$component}--back-to-top) {
 		@include c-pagination-bar-two-item-justify;
 	}
 
 	&__back-to-top {
-		display: none;
+		display: none !important; // Override button styles
```

Comment:
Kan niet opgelost worden door selector specifieker te maken?

Reply:
Is niet meer nodig, feedback van Tine geïmplementeerd